profiles/tests/test_filter_profiles.py

```diff
@@ -0,0 +1,212 @@
+from rest_framework.test import APITestCase, APIClient
+from authentication.models import CustomUser
+from profiles.models import Profile, Category, Activity
+
+
+def dump(response):
```

Comment:
I strongly recomment:
1. To extract this helper function into a separate file.
2. To include it (mandatory) with # noqa comment into each unittests file which deals with API.

Reply:
I removed it eventually to keep the code cleaner (forgot to do it before commit). But I'll add separate file with it then , thanks.